packages/@aws-cdk/aws-apigateway/lib/integration.ts

```diff
@@ -134,6 +134,20 @@ export interface IntegrationProps {
   readonly options?: IntegrationOptions;
 }
 
+/**
+ * Result of binding an Integration to the Method
+ */
+export interface IntegrationConfig {
+  /**
+   * This value is included in computing the Deployment's fingerprint. When the fingerprint
+   * changes, a new deployment is triggered.
+   * This property should contain values associated with the Integration that upon changing
+   * should trigger a fresh the Deployment needs to be refreshed.
+   * @default undefined deployments are not triggered for any change to this integration.
+   */
+  readonly deploymentToken?: { [key: string]: string };
```

Comment:
why complicate?  Make it a string. It's opaque, so making it a key/value sends the wrong message.